Fix windows test failures on master in @theia/ai-codex

<!-- Please provide a detailed description of the bug. -->
<!-- Note: This template is not meant for security vulnerabilities disclosure -->
<!-- Any such issue, created in this repo, will be deleted on sight -->
<!-- Instead please report vulnerabilities to the Eclipse Foundation's security team -->
<!-- For more details, please read SECURITY.md in the repository root -->
### Bug Description:

Follow up of GH-16484
@theia/ai-codex test case fails on windows runners

```sh
@theia/ai-codex:     Request Building
@theia/ai-codex:       1) should include workspace root in request
@theia/ai-codex:       ✔ should pass prompt to backend
@theia/ai-codex:       ✔ should pass sessionId to backend
@theia/ai-codex:       ✔ should merge custom options with defaults
@theia/ai-codex:   48 passing (126ms)
@theia/ai-codex:   1 failing
@theia/ai-codex:   1) CodexFrontendService
@theia/ai-codex:        Request Building
@theia/ai-codex:          should include workspace root in request:
@theia/ai-codex:      AssertionError: expected '\test\workspace' to include 'test/workspace'
@theia/ai-codex:       at Context.<anonymous> (lib\browser\codex-frontend-service.spec.js:168:123)
```

Investigate and fix the test to work cross-platform, as this PR was already merged to master.

<!-- Please provide clear steps to reproduce the bug. -->
### Steps to Reproduce:

https://github.com/eclipse-theia/theia/actions/runs/19483070968/job/55759014456

<!-- Please provide any additional information available. -->
<!-- Additional information can be in the form of logs, screenshots, screencasts. -->

### Additional Information

- Operating System: Windows
- Theia Version: >= https://github.com/eclipse-theia/theia/pull/16484

--- a/packages/ai-codex/src/browser/codex-frontend-service.spec.ts
+++ b/packages/ai-codex/src/browser/codex-frontend-service.spec.ts
@@ -18,6 +18,7 @@ import { enableJSDOM } from '@theia/core/lib/browser/test/jsdom';
 let disableJSDOM = enableJSDOM();
 import { FrontendApplicationConfigProvider } from '@theia/core/lib/browser/frontend-application-config-provider';
 FrontendApplicationConfigProvider.set({});
+import * as path from 'path';
 
 import { expect } from 'chai';
 import * as sinon from 'sinon';
@@ -206,7 +207,8 @@ describe('CodexFrontendService', () => {
             await service.send({ prompt: 'test prompt', sessionId: 'session-1' });
 
             const backendRequest = (mockBackendService.send as sinon.SinonStub).firstCall.args[0];
-            expect(backendRequest.options?.workingDirectory).to.include('test/workspace');
+            const expectedPath = path.join('test', 'workspace');
+            expect(backendRequest.options?.workingDirectory).to.include(expectedPath);
         });
 
         it('should pass prompt to backend', async () => {

